build.gradle

```diff
@@ -289,7 +289,6 @@ configurations.all {
 }
 
 dependencies {
-    implementation 'jakarta.annotation:jakarta.annotation-api:1.3.5'
```

Comment:
this change should already be part of 2.x via https://github.com/opensearch-project/security/pull/2803/files
but isn't